pkg/k8shandler/configmaps.go

```diff
@@ -46,6 +46,53 @@ type indexSettingsStruct struct {
 	ReplicaShards string
 }
 
+// CreateOrUpdateConfigMap reconciles a configmap
+func (elasticsearchRequest *ElasticsearchRequest) CreateOrUpdateConfigMap(cm *v1.ConfigMap) error {
+	dpl := elasticsearchRequest.cluster
+	err := elasticsearchRequest.client.Create(context.TODO(), cm)
+	if err == nil {
+		logrus.Info("created configmap", "name", cm.Name)
```

Comment:
can we change this to be at Debug level instead? Also can we include the namespace we're creating in since EO can modify configmaps in any namespace